### PR TITLE
enable content Preview - fixes #1383

### LINKED ISSUE #1383: [4.0.0-rc-1] Preview does not work
The preview of the content does not work.

### PR DESCRIPTION
**Pull request by emanueleg**

### CODE REVIEW

File: bl-plugins/api/plugin.php [hunk 1]
```diff
@@ -156,6 +156,8 @@ public function beforeAll()
                 $key = $parmC.'/'.$parmD;
             }
             $data = $this->getFiles($key);
+            //$page = new Page($pageKey);
```
Comment (dignajar) [inline bl-plugins/api/plugin.php:159]: Why add lines commented?

Comment (emanueleg) [reply bl-plugins/api/plugin.php:163]: My fault, it was a test and I forgot to remove these two lines before pushing

File: bl-plugins/api/plugin.php [hunk 2]
```diff
@@ -539,11 +541,14 @@ private function editPage($args)
                 'message'=>'Error trying to edit the page.'
             );
         }
+        
+        $page = new Page($newKey);
```
[the remark this answers is not in the document]
Comment (emanueleg) [reply bl-plugins/api/plugin.php:554]: it is used in the next line to get the uuid of the page and compute its md5 hash (needed for the preview)

**Comment by dignajar** [comment]
Thanks!

**Pull request merged**